### PR TITLE
Fix fast org policies

### PR DESCRIPTION
**Pull request by LucaPrete**

### CODE REVIEW

File: fast/stages/01-resman/organization.tf [hunk 1]
```diff
@@ -93,13 +93,13 @@ module "organization" {
...
-      local.list_deny, { values = ["ALLOW_INTERNAL_ONLY"] }
+      local.list_deny, { values = ["is:ALLOW_INTERNAL_ONLY"] }
     )
     "constraints/cloudfunctions.allowedVpcConnectorEgressSettings" = merge(
-      local.list_deny, { values = ["PRIVATE_RANGES_ONLY"] }
+      local.list_deny, { values = ["is:PRIVATE_RANGES_ONLY"] }
     )
     "constraints/compute.restrictLoadBalancerCreationForTypes" = merge(
-      local.list_deny, { values = ["in:INTERNAL"] }
+      local.list_deny, { values = ["in:EXTERNAL"] }
```
Comment (juliocc) [inline fast/stages/01-resman/organization.tf:102]: why the change to in:EXTERNAL?

Comment (sruffilli) [reply fast/stages/01-resman/organization.tf:102]: Goal is to deny the creation of XLBs and only allow ILBs.

Comment (LucaPrete) [reply fast/stages/01-resman/organization.tf]: This has been replaced with the local syntax now
